wgpu-hal/Cargo.toml

```diff
@@ -163,13 +163,16 @@ windows = { workspace = true, optional = true }
 bit-set = { workspace = true, optional = true }
 range-alloc = { workspace = true, optional = true }
 gpu-allocator = { workspace = true, optional = true }
-mach-dxcompiler-rs = { workspace = true, optional = true }
 # For core macros. This crate is also reexported as windows::core.
 windows-core = { workspace = true, optional = true }
 
 # backend: Gles
 glutin_wgl_sys = { workspace = true, optional = true }
 
+# This doesn't support aarch64. See https://github.com/gfx-rs/wgpu/issues/6860.
+[target.'cfg(all(windows, not(target_arch = "aarch64")))'.dependencies]
```

Comment:
thought: Might be worth adding a bi-directional link/note that this is intended to be kept in-sync with the `static-dxc` `cfg` alias?